doc/how-to/sql/improving_mysql.rst

```diff
@@ -279,59 +262,66 @@ So let’s proceed.
+
+#.  Next, start up the replicator:
+
+    ..  code-block:: console
 
-   .. code-block:: bash
+        $ systemctl start replicatord
 
-      cp replicatord.yml /usr/local/etc/replicatord.yml
+#.  Enter the Tarantool instance:
 
-#. Next we can start up the replicator:
+    ..  code-block:: bash
 
-   .. code-block:: bash
+        $ tt connect example
 
-      systemctl start replicatord
+#.  Do a select on the ``mysqldata`` space. The replicated content from MySQL looks the following way:
 
-   Now we can enter our Tarantool instance and do a select on the “mysqldata”
-   space. We will see the replicated content from MySQL:
+    ..  code-block:: tarantoolsession
 
-   .. code-block:: bash
+        tarantool> box.space.mysqldata:select()
+        ---
+        - - [1, 'Fluffy', 'Harold', 'cat']
+            - [2, 'Claws', 'Gwen', 'cat']
+            - [3, 'Buffy', 'Harold', 'dog']
+            - [4, 'Fang', 'Benny', 'dog']
+            - [5, 'Bowser', 'Diane', 'dog']
+            - [6, 'Chirpy', 'Gwen', 'bird']
+            - [7, 'Whistler', 'Gwen', 'bird']
+            - [8, 'Slim', 'Benny', 'snake']
+            - [9, 'Puffball', 'Diane', 'hamster']
 
-      tt connect example
+..  _improving_mysql-replicator:
 
-   .. code-block:: tarantoolsession
+Testing the replication
+-----------------------
 
-       tarantool> box.space.mysqldata:select()
-       ---
-       - - [1, 'Fluffy', 'Harold', 'cat']
-         - [2, 'Claws', 'Gwen', 'cat']
-         - [3, 'Buffy', 'Harold', 'dog']
-         - [4, 'Fang', 'Benny', 'dog']
-         - [5, 'Bowser', 'Diane', 'dog']
-         - [6, 'Chirpy', 'Gwen', 'bird']
-         - [7, 'Whistler', 'Gwen', 'bird']
-         - [8, 'Slim', 'Benny', 'snake']
-         - [9, 'Puffball', 'Diane', 'hamster']
+Finally, let’s enter a record into MySQL and then go back to Tarantool to make sure it’s replicated.
 
+#.  Exit the Tarantool instance with ``CTRL-C``.
 
-#. Finally let’s enter a record into MySQL and then go back to Tarantool to make
-   sure it’s replicated. So first we’ll exit our Tarantool instance with
-   ``CTRL-C``, and then say:
+#.  Insert a record into MySQL:
 
-   .. code-block:: sql
+    ..  code-block:: sql
 
-      mysql -u root -p
-      USE menagerie;
-      INSERT INTO pet(name2, owner, species) VALUES ('Spot', 'Brad', 'dog');
-      QUIT
+        mysql -u root -p
+        USE menagerie;
+        INSERT INTO pet(name2, owner, species) VALUES ('Spot', 'Brad', 'dog');
+        QUIT
 
-   Once back in the terminal enter:
+#.  In the terminal, enter the Tarantool instance:
 
-   .. code-block:: bash
+    ..  code-block:: bash
 
-      tt connect example
+        $ tt connect example
 
-   .. code-block:: tarantoolsession
+#   To see the replicated data in Tarantool, run the following command:
 
-      tarantool> box.space.mysqldata:select()
+    ..  code-block:: tarantoolsession
 
-   You should see the replicated data in Tarantool!
+        tarantool> box.space.mysqldata:select()
```

Comment:
Nit: This `#` is rendered as is (I guess it should be a number) and the code block is indented not like the previous one.